README.md

```diff
@@ -1,10 +1,12 @@
-# Whisk ML Project Framework
+[![whisk](/docs/img/whisk_logo.png)](https://github.com/whisk-ml/whisk)
```

Comment:
Most of the README files I see have the title of the project in text. I'd probably leave that there. The README is designed to be readable as a text file, and without an explicit header, it's not.

Reply:
The logo is super large. I'd make a smaller version about the size as when hovering over the org name and just use the logo, not the text.

![image](https://user-images.githubusercontent.com/7880/81934796-c75e9700-95ac-11ea-961f-1a274659cf91.png)

Reply:
Hm yeah so I did a quick survey before setting this up to see what other repo's did and how I liked the looks of them. All DS-related, so this might be a biased sample:

**Only logo:**
https://github.com/fishtown-analytics/dbt
https://github.com/bentoml/BentoML
https://github.com/tensorflow/tensorflow
https://github.com/pytorch/pytorch

**Text + Logo:**
https://github.com/keras-team/keras

**Text only**:
https://github.com/cortexlabs/cortex
https://github.com/drivendata/cookiecutter-data-science
https://github.com/apache/airflow
https://github.com/facebook/prophet

I liked the first section best (I thought DBT's was great) so I mimic'ed that one. None of the logo ones had small logos, it was either a major header or none at all. 

I'll give it a shot with a smaller logo without text, and a title at the top! We can see how that goes. I think a better looking logo might help make it more attractive to have the image large at the top too, ha